packages/calcite-components/src/components/rating/rating.scss

```diff
@@ -5,7 +5,8 @@
  *
  * @prop --calcite-rating-spacing-unit: [Deprecated] Use `--calcite-rating-spacing`. Specifies the amount of left and right margin spacing between each item.
  * @prop --calcite-rating-spacing: Specifies the amount of left and right margin spacing between each item.
- * @prop --calcite-rating-color-hover: Specifies the component's item color when hovered or selected.
+ * @prop --calcite-rating-color-hover: Specifies the component's item color when hovered.
+ * @prop --calcite-rating-color-press: Specifies the component's item color when active.
```

Comment:
+@geospatialem @DitwanP @matgalla for CSS prop doc review.

Reply:
LGTM and in alignment with other component styles ⭐🚀⭐